src/app/components/test-card-detailed/test-card-detailed.component.html

```diff
@@ -107,26 +105,94 @@
 
       <div class="pl-10 text-xl">
 
+
         <div class="flex pt-5">
-          <div class="hover:scale-110 duration-700">
-            <svg-icon src="./assets/facebook-rounded-svgrepo-com.svg"
-                      [svgStyle]="{ 'width.px':50, 'height.px':50, 'fill':'rgb(0,0,0)' }"></svg-icon>
+
+
```

Comment:
Please remove all unnesecary empty lines. Related to all files. There might be emty lines, but only to make code more readable and no more than 1 in a row

Reply:
removed all unnesecary empty lines

Reply:
done